USAGE_android.md

```diff
@@ -19,101 +19,70 @@ to one of these other documents:
  * [GFXReconstruct for Desktop Vulkan](./USAGE_desktop_Vulkan.md)
  * [GFXReconstruct for Desktop D3D12](./USAGE_desktop_D3D12.md)
 
-
 ## Index
 
-1. [Behavior on Android](#behavior-on-android)
-    1. [Android Writable Locations](#android-writable-locations)
-2. [Capturing API Calls](#capturing-api-calls)
+1. [Capturing API Calls](#capturing-api-calls)
     1. [Before Use](#before-use)
-    2. [To Root or Not To Root](#to-root-or-not-to-root)
-    2. [Enabling the Layer with ADB](#enabling-the-layer-with-adb)
+    2. [Enabling the Capture Layer](#enabling-the-capture-layer)
     3. [Capture Options](#capture-options)
     4. [Capture Files](#capture-files)
     6. [Capture Limitations](#capture-limitations)
     7. [Troubleshooting Capturing of Applications](#troubleshooting-capturing-of-applications)
-3. [Replaying API Calls](#replaying-api-calls)
+2. [Replaying API Calls](#replaying-api-calls)
     1. [Launch Script](#launch-script)
     2. [Install APK Command](#install-apk-command)
     3. [Replay Command](#replay-command)
     4. [Touch Controls](#touch-controls)
     5. [Key Controls](#key-controls)
     6. [Limitations of Replay On Android](#limitations-of-replay-on-android)
     7. [Troubleshooting Replay of Applications](#troubleshooting-replay-of-applications)
-4. [Android Detailed Examples](#android-detailed-examples)
-
-
-## Behavior on Android
-
-The purpose of this section is to describe some of the software changes made to
-the GFXReconstruct software to add Android support.
-This section will not provide a comprehensive list of changes, but will instead
-highlight some of the primary adjustments required to adapt the GFXReconstruct
-software to the Android ecosystem.
-
-### Android Writable Locations
-
-The contents of the traces should be written to external storage on the
-Android device.
-The final "external storage" result varies based on Android version but some
-locations that can be tried are:
-
- - `/sdcard/Download`
- - `/storage/emulated/0/Download`
- - `/sdcard/Android/data/${Application Full Name}`
- - `/sdcard`
- - `/mnt/shell/emulated/0`
-
```

Comment:
Isn't this still needed for release versions of an application?  It still might be your own application showing different behavior in a release build with no standard mechanism.

Reply:
I kept the writable location section but moved it under the Capturing API Calls > Before Use section below. I also tweaked it a bit to emphasize the `/sdcard/Android/data/${Package Name}` path since it seemed to be the most reliable on both the user and userdbg builds I was testing.